README.md

```diff
@@ -1,24 +1,26 @@
 # Conduit Connector Google Cloud Storage
 
 ### General
-The Google Cloud Storage connector is one of [Conduit](https://github.com/ConduitIO/conduit) plugins. It provides a source GCS connectors.
+The Google Cloud Storage connector is one of [Conduit](https://github.com/ConduitIO/conduit) plugins. It provides source GCS connectors.
 
 ### How to build it
 Run `make`.
 
 ### Testing
 
-Run `make test` to run all the tests. You must set the environment variables (`GCP_ServiceAccount_Key`
+Run `make test` with optional `GOTEST_FLAGS` set to run all the tests. You must set the environment variables (`GCP_ServiceAccount_Key`
 , `GCP_ProjectID`, `GCP_Bucket`)
 before you run all the tests. If not set, the tests that use these variables will be ignored.
 
+Cases/Scenarios which are dependent on GCS Error response and where these can't be reproducible are ignored in the test cases.
+ 
+Integration test cases are written to cover all of the scenarios of snapshot and cdc iterators except when tomb dead as it happens only in the above mentioned case.
+ 
 ## GCS Source
 
-The Google Cloud Storage Source Connector connects to a GCS bucket with the provided configurations, using serviceAccountKey,bucket details . Then will call `Configure` to parse the
-configurations and make sure the bucket exists, If the bucket doesn't exist, or the permissions fail, then an error will
-occur. After that, the
-`Open` method is called to start the connection from the provided position.
-
+The Google Cloud Storage Source Connector connects to a GCS bucket with the provided configurations, using serviceAccountKey(`JSON`) and bucket(`BucketName`) details . Then will call `Configure` to parse the
```

Comment:
JSON here is confusing.. it's a format, it's not the value nor the key itself
```suggestion
The Google Cloud Storage Source Connector connects to a GCS bucket with the provided configurations, using `serviceAccountKey` and `bucket` details from the configurations. Then will call `Configure` to parse the
```

Reply:
Done